.svglintrc.js

```diff
@@ -473,6 +473,43 @@ module.exports = {
                 ignoreIcon(reporter.name, iconPath, $);
               }
             }
+          },
+          function(reporter, $, ast) {
+            reporter.name = "path-format";
+
+            const iconPath = $.find("path").attr("d");
+            if (!updateIgnoreFile && isIgnored(reporter.name, iconPath)) {
+              return;
+            }
+
+            const validPathFormatRegex = /^[Mm][MmZzLlHhVvCcSsQqTtAaEe0-9-,.\s]+$/;
+            if (iconPath.match(validPathFormatRegex) === null) {
```

Comment:
I would suggest using this instead

```suggestion
            if (validPathFormatRegex.test(iconPath)) {
```

To avoid an error if `iconPath` is somehow not a string.